File: C/Basic Maths/evenOdd.c
#include <stdio.h>

// program to print n even numbers
void main() {
    int num, i;
    printf("Enter a number: ");
    scanf("%d", &num);
    printf("Even numbers are: \n");
    for (i=0; i <=num; i+=2) {
        printf("%d\n", i);    
    }
}

//or

void main() {
    int num, i;
    printf("Enter a number: ");
    scanf("%d", &num);
    printf("Even numbers are: \n");
    for (i=0; i <=num; i++) {
        if (i%2 == 0) {
            printf("%d\n", i);
        }
    }
}


//============================================================================
//============================================================================


// program to print n odd numbers
void main() {
    int num, i;
    printf("Enter a number: ");
    scanf("%d", &num);
    printf("Odd numbers are: \n");
    for (i=1; i <=num; i+=2) {
        printf("%d\n", i);    
    }
}

//or

void main() {
    int num, i;
    printf("Enter a number: ");
    scanf("%d", &num);
    printf("Odd numbers are: \n");
    for (i=0; i <=num; i++) {
        if (i%2 != 0) {
            printf("%d\n", i);
        }
    }
}


//============================================================================
//============================================================================


// program to find a number is even or odd
void main() {
    int num;
    printf("Enter a number: ");
    scanf("%d", &num);
    if (num%2 == 0) {
        printf("%d is Even.", num);
    } else {
        printf("%d is Odd.", num);
    }
}


//============================================================================
//============================================================================

File: Python/Basic Maths/evenOdd.py
# program to print n even numbers
num = int(input("Enter a number: "))
print("Even numbers are: ")
for x in range(0, num+1):
    if x%2 == 0:
        print(x)
        

#============================================================================
#============================================================================


# program to print n odd numbers
num = int(input("Enter a number: "))
print("Odd numbers are: ")
for x in range(0, num+1):
    if x%2 != 0:
        print(x)
          

#============================================================================
#============================================================================


# program to check a number is even or odd
num = int(input("Enter a number: "))
if num%2 == 0:
    print(f"{num} is Even.")
else:
    print(f"{num} is Odd.")
    